Open Browser after HTML is generated

nscyclone opened the pull request:
Closes #1. I am not sure what exactly was implied in [the original issue](https://github.com/PragTob/benchee_html/issues/1), but I think that this could be considered a good start.
It would also be great to make this PR a part of [the Hacktoberfest challenge](https://hacktoberfest.digitalocean.com/).

devonestes commented:
Thanks for the contribution! I think it will be part of Hacktoberfest as long as you've signed up. They track all the PRs you open for the month (or at least they did last year). The tags on issues and such are just helpers for folks to find stuff to do.

So this seems like a good way to go for me, but I'd love to see some sort of test for this new behavior. Think you could find a way to add something to test this? It'll be a little tricky because it's dependent on the OS on which the test is run, but with a little creativity I'm sure you can find a design that allows you to test this stuff.

nscyclone commented:
Thanks for detailed response, @devonestes! I've just reworked `open_report/1` to make it accept `suite` instead of `url` and updated the test for `output/1` with a very simple check.

PragTob commented:
:wave: 

Yup, some user facing configuration would be nice. If it breaks, as it'll be super hard to make it work on every OS and I don't just wanna error out in all those cases. So a user option might be nice. Not sure what it should default to, I guess if it works on all our machines we could make it the default and wait for the bug reports to pile in :)
